### DISCUSSION TITLE
Support for Super Princess Peach sequences

### DISCUSSION THREAD

**Discussion by Alexkaruso** [vgmtrans/vgmtrans#766] [Music formats]
Maybe some of you can help me  

I have the following problem when I try to open super princes peach vgmtrans ignores it completely, it doesn't load anything but the program don't freeze it only won't load the NDS file  I've already tried all 3 Versions of the game, but all do the same one thing: nothing

Maybe one of you has an Idea what can solve my problem

**Comment by sykhro** [comment]
The sequence format of super princess peach is currently unsupported.

**Comment by sykhro** [comment]
As per recent decision, we don't take feature requests for formats. Moving to discussions
